src_assets/windows/misc/gamepad/install-gamepad.bat

```diff
@@ -1,6 +1,13 @@
 @echo off
 setlocal enabledelayedexpansion
 
+rem Check if a compatible version of ViGEmBus is already installed (1.17 or later)
+set Version=
+for /f "usebackq delims=" %%a in (`"wmic product where name='ViGEm Bus Driver' get Version /format:Textvaluelist"`) do (
+    for /f "delims=" %%# in ("%%a") do set "%%#"
+)
+if /i %Version:~2,4% geq 17 exit /b 0
+
```

Comment:
```suggestion
rem Check if a compatible version of ViGEmBus is already installed (1.17 or later)
set Version=
for /f "usebackq delims=" %%a in (`"wmic product where name='ViGEm Bus Driver' get Version /format:Textvaluelist"`) do (
    for /f "delims=" %%# in ("%%a") do set "%%#"
)

rem Extract Major and Minor versions
for /f "tokens=1,2 delims=." %%a in ("%Version%") do (
    set "MajorVersion=%%a"
    set "MinorVersion=%%b"
)

rem Compare the version to 1.17
if /i !MajorVersion! gtr 1 goto skip
if /i !MajorVersion! equ 1 (
    if /i !MinorVersion! geq 17 (
        goto skip
    )
)
goto continue

:skip
echo "The installed version is %Version%, no update needed. Exiting."
exit /b 0

:continue
```

A few tweaks.

- Parse semver major and minor versions (not relying on number of characters in the version.
- Compare the major version first, skip if installed version is greater (although highly unlikely with vigembus)
- Compare the minor version after comparing the major version.